tgs/src/main/cpp/tgs/BigContainers/BigMap.h

```diff
@@ -22,22 +22,118 @@
  * This will properly maintain the copyright information. DigitalGlobe
  * copyrights will be updated automatically.
  *
- * @copyright Copyright (C) 2015 DigitalGlobe (http://www.digitalglobe.com/)
+ * @copyright Copyright (C) 2015, 2017 DigitalGlobe (http://www.digitalglobe.com/)
  */
-#ifndef BIGMAP_H
-#define BIGMAP_H
+#ifndef BIGMAPHYBRID_H
+#define BIGMAPHYBRID_H
```

Comment:
I actually deleted the contents of `BigMap.h` and renamed `BigMapHybrid.h` and this diff is what I got.  I made no changes to the file.